### PR TITLE
Colonial Health Station (and a medipen)

### PR DESCRIPTION
**Pull request by Stalkeros2**
## About The Pull Request
Adds colonial health stations and accompanying under-/over-/godknowshow-powered (I suck at chemicals) medipens. Also a new company for all of the colonial items but that doesn't really matter.
image
image
image
Said medipens aren't that special, being effectively a toned down version of mining ones.
As for the health stations, they posses a couple relatively unique functions, being the ability to receive health and chemical scans; treat one random wound of yours; treat a half of your total burn and brute damage; and refill most of the generic medipens and the whole two proprietary medipens (EVA glucose and colonial survival ones). Refillable with mining monster organs, ambrosia gaia and RnD regenerative slime extracts.

## How This Contributes To The Nova Sector Roleplay Experience
~~I felt like it.~~ 
Initially, this was meant to be a replacement of those funny health stations (reskinned IV drips) you see in the MilSim domain; a more fleshed out, better looking one. Still remains that, given that they were a placeholder.
As for the station implementation, I reckon that it wouldn't hurt anyone, *literally*, by having something like this. Stationary machines applying medicine/healing, in general, are an amazing concept that sadly wasn't really expanded upon here.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  Screenshots above with everything working like it should.
</details>

## Changelog

:cl: Stalkeros, Zydras for the sprites, Juniper for health station name and description
add: Colonial health stations with a funny name, available for purchase from the "Surplus" Military Surplus Company.
add: Accompanying colonial survival medipen, also buyable from the same company.
/:cl:

### CODE REVIEW

File: modular_nova/modules/health_station/code/health_pen.dm [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	icon_state = "pen"
+	inhand_icon_state = "stimpen"
+	base_icon_state = "pen"
+	volume = 30
+	amount_per_transfer_from_this = 30
+	list_reagents = list(/datum/reagent/medicine/oxandrolone = 10,
+	/datum/reagent/medicine/sal_acid = 10,
+	/datum/reagent/medicine/salbutamol = 5,
+	/datum/reagent/medicine/morphine = 5,
+	)
```
Comment (Zergspower) [inline modular_nova/modules/health_station/code/health_pen.dm:14]: this is far too overtuned for 100 credits, that's both pain numbing, enough chems to health 150 burn and brute on top of oxy loss coverage

Comment (Stalkeros2) [reply modular_nova/modules/health_station/code/health_pen.dm:14]: > this is far too overtuned for 100 credits, that's both pain numbing, enough chems to heal both 150 burn and brute on top of oxy loss coverage

oh
brb

**Comment by HoodMilk** [comment]
only if they make the half life health charger sounds

**Comment by Stalkeros2** [comment]
> only if they make the half life health charger sounds

They're copyrighted or something so sadly I can't.

**Comment by chemistrymain2** [comment]
I would prefer not to move treatment and healing options to uninteractive non-player based ones, _especially_ with the 'more things to add to cargo' list.

Medbay often has little enough work to do and moving people away from this- especially with, yet again, more content added _to cargo_ at the expense of other departments who have minimal interaction with it- is not great.

Science doesn't get any new toys, this is the _opposite_ of a new toy for the medbay, etc- I'd really rather not have automatic heal stations that can treat wounds orderable from cargo. Not for a 'this will be overpowered and abused' sense but from a 'this doesn't really add anything to the game' side of things- it's just kind of meh.

It's not going to ruin the game for medbay or anything but i'd rather that sort of functionality being added to something medbay _uses_ as opposed to like a health pickup.

**Comment by chemistrymain2** [comment]
If you made medbay restock these even it'd be a lot better.

**Comment by Stalkeros2** [comment]
> I would prefer not to move treatment and healing options to uninteractive non-player based ones, _especially_ with the 'more things to add to cargo' list.

(Sorry for the late af response, I sincerely thought I've responded to all that until you've sent the second message :skull:) It's not -that- reliant on Cargo; excluding the shipping part (to be honest, how else would you procure one), currently, you have to request botany (most likely, as it's the most efficient and replenishable way of them all) for the refills. There's mining and science too with their regenerative cores, but I doubt anyone'll collect them for the machines outside of the initial merge, (since the former's cores are pretty tedious to collect, and latter's are too good to be recycled by one) if it does get in.
> Medbay often has little enough work to do and moving people away from this- especially with, yet again, more content added _to cargo_ at the expense of other departments who have minimal interaction with it- is not great.

Only Cargo interaction is getting the machine; everything else is reliant on other departments. And it already has some really long delays for its usage, to the point where it'll genuinely be faster to be treated by someone else.

> Science doesn't get any new toys, this is the _opposite_ of a new toy for the medbay, etc- I'd really rather not have automatic heal stations that can treat wounds orderable from cargo. Not for a 'this will be overpowered and abused' sense but from a 'this doesn't really add anything to the game' side of things- it's just kind of meh. It's not going to ruin the game for medbay or anything but i'd rather that sort of functionality being added to something medbay _uses_ as opposed to like a health pickup.

That's not entirely true, as it lets medical to refill their used medipens that are always available roundstart. All of the healing methods are near useless unless you're effectively near-dead from physical damages (tend wounds halving all your total burn-brute damage and thus being progressively less efficient the less you're damaged).

**Comment by Stalkeros2** [comment]
> If you made medbay restock these even it'd be a lot better.

I don't think medical produces anything- Actually I can just add an item to do that.

**Review by Jolly-66 — APPROVED** [review]
Map changes are fine.

**Comment by AdamElTablawy** [comment]
I’m just going to be honest dude it is boring as fuck in medbay right now and handing over what little job content I still have to an automated mechanicsjak machine does not fill me with enthusiasm

**Comment by chemistrymain2** [comment]
Longer post:

I genuinely cannot get across how much I actually despise this change.

It's not a medbay-interactive device or tool. It's not even a tool that particularly relies on medbay- it 'relies' on 'interdepartmental cooperation' from other departments in the same way that some other facets rely on poorly integrated 'eh i guess' sort of off-use cases. It's effectively random, annoying-to-get to make something work, akin to anomaly cores.

Except at least even anomaly cores are thematic and gives mild responsibility to science. This doesn't.

This is an item, ultimately, that basically exists to take work away from the medbay. It's not like the DeForest kits, which are thematic, and can be used by the medbay themselves and exist as medbay tools- it's an automated wound-tender.

In a gamestate where medbay already sees very few people and is harassed by people for 'just fix me already stop talking' and where most of our interactions boil down to 'stasis bed, tend wounds, every problem fixed' and there's no complexity or involvement at all, where we already have functionally infinite supplies of every single superhealing chemical, et cetera- we already have a dearth of job content.

Creating an item that _everyone uses but medical_ that takes work away from medical is perhaps the last thing medbay needs. It's not like an automated resin foam turret or something that engineering could get to seal breaches- it's just a port-a-doc that shittily fixes problems.

It's nerfed into uselessness, of course. But that just means someone will look at it, go 'this item sucks', make it easier to use, and then it becomes a nuisance- or it stays as a useless item, which means why does it exist at all?

It's not a tool the medbay uses. It's just...job theft.

**Pull request closed without merging**